app/components/partners/partner.js

```diff
@@ -2,12 +2,12 @@ import Link from 'next/link';
 import { mediaQueries } from '../../constants';
 
 const Partner = ({
-  href, className, img, title,
+  href, className, img, title, year,
 }) => (
   <li>
     <Link href={`${href}`}>
       <a className={className} target="_blank">
-        <img src={`static/img/partners/${img}`} title={`${title}`} alt={`${title}`} />
+        <img src={`static/img/partners/${year}/${img}`} title={`${title}`} alt={`${title}`} />
```

Comment:
wouldn't it work better if we construct the base url just once / component based on year? also maybe having a 2017, 2018 root folder and then inside each year to replicate the structure: `partners`, `speakers`, etc.